CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+- Support for generating `lamdera/program-test` `Effect.Http` requests.
```

Comment:
Would this differ at all between someone wanting to use `avh4/elm-program-test` and `lamdera/program-test`? I think so but want to make sure. Asking because I'm wondering if the effect type should be the full package name. Something like
```
elm/http.cmd
elm/http.cmdrisky
elm/http.task
elm/http.taskrisky
dillonkearns/elm-pages.backendtask
lamdera/program-test.programtest
lamdera/program-test.programtesttask
```

those are verbose, but also more clear as to where they're coming from, especially if we added say `avh4/elm-program-test` because it'd be more difficult to tell that apart from the Lamdera one.


Would love to get your thoughts

Reply:
Yeah, I'm in favor of long package name honestly.
I think `elm/http` can keep the short ones, because that is going to be the most common.

For non-elm/http we could just always have all kinds?

So we'd have:
```
cmd
cmd-risky
task
task-risky
dillonkearns/elm-pages
lamdera/program-test
[in the future] avh4/elm-program-test
```

Reply:
I think that's definitely an improvement! And keeping the basics, basic, makes a lot of sense

Reply:
Ooooor, we could split it all!

```
--types:  a list of (cmd | task | cmdrisky | taskrisky)
--packages: a list of (elm/http | lamdera/program-test | dillonkearns/elm-pages) - if empty use [elm/http]
```

Reply:
Hmm, the second approach is kind of appealing. And I don't foresee people mixing packages too much, if at all. E.g. if I were to want to generate both and SDK and a test SDK I think I might have 2 separate commands. One would generate the production/development SDK and the other would generate the test SDK. I'd say there's a limited chance that someone will want/need to generate for say `elm/http` & `dillonkearns/elm-pages` in the same script.

Reply:
I think maybe my 1 hesitation with the second approach is `--packages dillonkearns/elm-pages` doesn't have a matching `--types ???` because it's a separate `backendtask`. Similar if/when support for `ConcurrentTask` is added, there's no matching cmd/task type.

Reply:
Moved this to an issue so it can be separate work https://github.com/wolfadex/elm-open-api-cli/issues/110